src/index.ts

```diff
@@ -11,6 +12,7 @@ async function main() {
   const stampConfig = getStampsConfig(process.env as EnvironmentVariables)
   const appConfig = getAppConfig(process.env as EnvironmentVariables)
   const { hostname, port } = getServerConfig(process.env as EnvironmentVariables)
+  startIdentityBackgroundJob()
```

Comment:
Hmm, I don't understand why this background job is needed. I mean the identity does not change or does it?

Reply:
It may not be available immediately - that is why it is not a one-time thing (e.g. Bee already reports healthy but is still starting up and some endpoints are 404).

But in this case, it would be better to retry until we get a sensible response instead of an interval running forever, so I will refactor to that.

Reply:
I see. Hmm, you can even keep the interval (but I would change the name for the function), but just stop the interval when you finally get the response. Also please add description to the function why it is done like this.